docs/contributing.md

```diff
@@ -7,7 +7,23 @@ custom_js_with_timestamps:
 - contributing.js
 ---
 
-{% include babel/CONTRIBUTING.md %}
```

Comment:
I would prefer keeping the documentation in Babel/babel repo. We can still update the file there.